vtex/styleguide
# Add TrapFocus component in Modal and fix Input autoFocus

#### What is the purpose of this pull request?
Implement our own solution to trap focus in Modal.

#### What problem is this solving?
Dialogs must have focus trapped inside.



#### How should this be manually tested?
`yarn && yarn storybook`

1. Go to http://localhost:6006/?path=/story/components-modal--default
2. Navigate through Modal with `TAB` and `SHIFT+TAB` and check if it is trapped inside modal.

#### Types of changes
